siriuspy/requirements.txt

```diff
@@ -2,7 +2,6 @@ aiohttp>=3.7.4
 bottleneck>=1.3.2
 mathphys
 numpy<=1.23
-pyepics>=3.4.0
```

Comment:
I prefer to keep this requirement

Reply:
So in your environment, you install only those interesting requirements, and do not use this file for that